telco-hub/configuration/reference-crs/required/storage/odf-internal/storageCLuster.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: ocs.openshift.io/v1
+kind: StorageCluster
+metadata:
+ name: ocs-storagecluster
+ namespace: openshift-storage
+spec:
+ manageNodes: false
+ resources:
+   mds:
+     limits:
+       cpu: "3"
+       memory: "8Gi"
+     requests:
+       cpu: "3"
+       memory: "8Gi"
+ monDataDirHostPath: /var/lib/rook
+ storageDeviceSets:
+ - count: 1  # <-- Modify count to desired value. For each set of 3 disks increment the count by 1.
+   dataPVCTemplate:
+     spec:
+       accessModes:
+       - ReadWriteOnce
+       resources:
+         requests:
+           storage: "600Gi"  # <-- This should be changed as per storage size. Minimum 100 GiB and Maximum 4 TiB
+       storageClassName: localblock  # match this with the storage block created at the LSO step
+       volumeMode: Block
+   name: ocs-deviceset
+   placement: {}
+   portable: false
+   replica: 3
+   resources:
+     limits:
+       cpu: "2"
+       memory: "5Gi"
+     requests:
+       cpu: "2"
+       memory: "5Gi"
```

Comment:
Are these values sufficient? In performance testing we increased these to requests 3cpu/8G and limits 10cpu/24G. We likely don't need that much for the hub but are these limits too low?